sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+//
+
```

Comment:
Is this decoration is necessary?

Reply:
@fredoh9 I think it isn't about decoration. The "C++"-style SPDX-License comment at the top of the file is an exception - see Documentation/process/license-rules.rst:
>    The SPDX license identifier is added in form of a comment.  The comment style depends on the file type::
> 
>       C source:	// SPDX-License-Identifier: <SPDX License Expression>
>       C header:	/* SPDX-License-Identifier: <SPDX License Expression> */
>       ASM:	/* SPDX-License-Identifier: <SPDX License Expression> */
>       scripts:	# SPDX-License-Identifier: <SPDX License Expression>
>       .rst:	.. SPDX-License-Identifier: <SPDX License Expression>
>       .dts{i}:	// SPDX-License-Identifier: <SPDX License Expression>

Otherwise any multi-line comments in the kernel should use `/* */`